orbstation/code/antagonists/rulesets_midround.dm

```diff
@@ -141,3 +141,48 @@
 	new_heretic.knowledge_points = min(new_heretic.knowledge_points, 5)
 
 	return TRUE
+
+//////////////////////////////////////////////
+//                                          //
+//              LONE OPERATIVE              //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/midround/from_ghosts/lone_operative
+	name = "Lone Operative"
+	midround_ruleset_style = MIDROUND_RULESET_STYLE_HEAVY
+	antag_datum = /datum/antagonist/nukeop/lone
+	antag_flag = ROLE_LONE_OPERATIVE
+	antag_flag_override = ROLE_NUCLEAR_OPERATIVE
+	required_enemies = list(2,2,2,2,1,1,1,0,0,0)
```

Comment:
This will probably make it _less_ common than now as it can only roll in rounds with two security or a captain, but that's fine as far as I am concerned.

Reply:
I considered this and decided nukies that don't have any serious opposition probably shouldn't appear. For now, at least!